json_object.c

```diff
@@ -167,8 +169,11 @@ static int json_escape_str(struct printbuf *pb, const char *str, int len, int fl
 extern struct json_object* json_object_get(struct json_object *jso)
 {
 	if (jso)
+	{
 		jso->_ref_count++;
-	return jso;
+		return jso;
+	}
+	return NULL;
```

Comment:
Without this commit, the value of jso was always read for the if and the return. With it, if with the first read it is NULL, it is not read a second time. Of course, that is the case for a stupid compiler that is not enough clever to optimize it-self.